### PR TITLE
8.0 split payment and fatturaPA

### PR DESCRIPTION
**Pull request by eLBati**
====================
Italy: split payment
====================

Law: http://goo.gl/1riAwt (Articolo 17 ter)

Module to generate Split Payment accounting entries

### CODE REVIEW

File: l10n_it_split_payment/README.rst [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/122/8.0
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/l10n-italy/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
+`here <https://github.com/OCA/l10n-italy/issues/new?body=module:%20l10n_it_split_payment%0Aversion:%20{version}%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```
Comment (tafaRU) [inline l10n_it_split_payment/README.rst]: Please replace ```{version}``` by 8.0

File: l10n_it_split_payment/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{
+    'name': 'Split Payment',
+    'version': '8.1.1.0.0',
```
Comment (tafaRU) [inline l10n_it_split_payment/__openerp__.py]: Should not be 8.0?

**Comment by dcorio** [comment]
@eLBati i'm quite sure that didn't happen to us. probably it's something related to some tax configuration or so.
anyway. for me it's ok to remove that from the official module.

**Comment by eLBati** [comment]
@tafaRU thanks, fixed

File: l10n_it_split_payment/README.rst [hunk 3]
```diff
@@ -0,0 +1,61 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+====================
+Italy: split payment
```
Comment (tafaRU) [inline l10n_it_split_payment/README.rst]: Would be better something like **Italian Localization - Split Payment**

Comment (eLBati) [inline l10n_it_split_payment/README.rst]: On 8 September 2015 at 15:48, Alex Comba <notifications@github.com> wrote:

> Would be better something like *Italian Localization - Split Payment*
>
>
>
Thanks, fixed

**Comment by tafaRU** [comment]
@eLBati please see https://github.com/eLBati/l10n-italy/pull/75

**Comment by eLBati** [comment]
On 16 September 2015 at 17:44, Alex Comba <notifications@github.com> wrote:

> @eLBati <https://github.com/eLBati> please see eLBati#75
> <https://github.com/eLBati/l10n-italy/pull/75>
>
>
>
Merged. Thanks

**Comment by dcorio** [comment]
@eLBati @tafaRU those 2 configuration fields in Settings > Accounting shouldn't be mandatory. otherwise you cannot select the chart of accounts (or other settings) if you installed the l10n_it_split_payment module first.

**Comment by eLBati** [comment]
@dcorio Thanks. Fixed with https://github.com/eLBati/l10n-italy/commit/89f70addfd970f29dcd46c6f47a72b961c5ad5b5

**Comment by andrea4ever** [comment]
:+1:

**Comment by OpenCode** [comment]
:+1:

**Pull request merged**